lib/jsdom/browser/Window.js

```diff
@@ -93,12 +94,11 @@ function Window(options) {
     }
   });
   // https://html.spec.whatwg.org/#session-history
-  this._sessionHistory = [{
+  this._sessionHistory = options.sessionHistory || new SessionHistory({
```

Comment:
This is intended as an internal option.  It'll be used when we do real navigation to create a new window with the same session history as an existing window.  Do you think there's a better way of expressing that?